### DISCUSSION TITLE
New Milestone Categories

### DISCUSSION THREAD

**Discussion by Archmonger** [idom-team/idom#895] [General]
With our upcoming switch to [semantic versioning](https://semver.org/), I have modified our milestone categories.

[Experimental/Theoretical](https://github.com/idom-team/idom/milestone/3)
Issues that are potentially not technologically feasible, or would require a lot of engineering to accomplish.
[Luxury](https://github.com/idom-team/idom/milestone/2)
Issues that would make developing in IDOM easier, but have pre-existing workarounds.
[Essential](https://github.com/idom-team/idom/milestone/4)
Issues that have an existing impact on the user's capability to develop.

cc: @rmorshea

**Comment by rmorshea** [comment]
I'm good with these. With that said, do you think we should make the first release of `reactpy` 1.0 or continue with 0 versioning for a while longer there.

**Reply by Archmonger** [reply]
I think it should be 1.0 to signify a production/stable release.

**Comment by rmorshea** [comment]
Renamed the last one to "Speculative"

**Comment by rmorshea** [comment]
Now that I'm thinking about it, these categories don't really seem any different from our priority labels.

**Reply by rmorshea** [reply]
Perhaps we should just create milestones for the next foreseeable versions. This might also be useful since we'll want to decouple the js and py package releases in order to allow them to evolve independently.

**Reply by rmorshea** [reply]
I created `release-*` labels for patch/minor/major. We can use these to organize things into genuine version milestones as we go.

**Reply by Archmonger** [reply]
We might want to take a look at some bigger projects and see how they organize their GitHub milestones and labels.
